web-client/src/app/views/delete-user/delete-user.page.ts

```diff
@@ -40,18 +44,47 @@ export class DeleteUserPage implements OnInit {
   /** @see validatedAddress */
   address?: string;
 
+  hideXrpBalance = environment.hideXrpBalance;
+
+  tokenIssuer = environment.tokenIssuer;
+
+  tokenSign: string | undefined;
+
+  /** A balance held by the current user. */
+  balances = this.sessionQuery.allBalances;
+
+  paymentOptions?: PaymentOption[];
```

Comment:
is this used anywhere?

Reply:
`tokenSign `is used in [delete-user.page.html](https://github.com/ntls-io/nautilus-wallet/blob/aaef347fe516a445a01d9ed6e577beed5afd5441/web-client/src/app/views/delete-user/delete-user.page.html#L13). However `balances` and paymentOptions are not.